scripts/run-ci-e2e-tests.js

```diff
@@ -70,6 +72,25 @@ try {
 
   const REACT_NATIVE_PACKAGE = path.join(ROOT, 'react-native-*.tgz');
 
+  describe('Verdaccio');
+  const verdaccioProcess = child_process.spawn('npx', [
+    'verdaccio',
+    '--config',
+    './.circleci/verdaccio/config.yml',
+  ]);
+  VERDACCIO_PID = verdaccioProcess.pid;
+  exec('npm set registry http://localhost:4873');
+
+  exec('node ./scripts/wait-for-verdaccio.js');
+
+  echo('Publish packages');
+  const packages = fs.readdirSync('packages');
+  packages.forEach(current_package => {
```

Comment:
This is not safe, I think it should at a minimum check that it's a folder etc, but preferably go after the yarn workspace config. You  can access it through `yarn workspaces info`